Xamarin.Forms.Platform.Android/Cells/TextCellRenderer.cs

```diff
@@ -29,6 +29,11 @@ protected override AView GetCellCore(Cell item, AView convertView, ViewGroup par
 
 		protected override void OnCellPropertyChanged(object sender, PropertyChangedEventArgs args)
 		{
+			if (View.IsDisposed())
```

Comment:
So, I know when chatting you had mentioned you wanted to make this method call as light as possible because how often PropertyChanged is called.

That being said then should we try only calling this for relevant properties?

I realize the code for that is a bit heavy but it would let us check for a few more things without the performance cost. Maybe the method calls at each line could use an action of sorts to get called

```C#
CallPropertyChangedMethod(UpdateMainText, this);
```

And then CallPropertyChangedMethod would check `this` to see if it should still call `UpdateMainText` and then at that point the check and any additional behavior could be generalized.